namui-cli/electron/src/main.js

```diff
@@ -30,8 +30,8 @@ function createWindow() {
 }
 
 app.whenReady().then(() => {
+    ipcMain.handle("config", () => config);
```

Comment:
`preload.js` will run on rederer process.
But option argv will be passed to main process.
So `getConfig()` should run on main.js and then send it through ipc.